### DISCUSSION TITLE
关于原有试验性API的存留与否

### DISCUSSION THREAD

**Discussion by yyuueexxiinngg** [howmanybots/onebot#134] [想法 / 建议]
目前已经存在项目使用试验性API, 如此[OtterBot Issue](https://github.com/Bluefissure/OtterBot/issues/70)便使用了`_get_friend_list`, 在协议项目发起后这些API时候会保留呢? 谢谢

**Comment by stdrc** [comment]
这个我觉得可以直接把新功能（和满足原先试验性接口的功能）实现为「公开 API」，旧的试验性接口应该不用兼容了

毕竟当时搞这个试验性 API 只是因为我在 CQHTTP 拿 cookie 调 web 接口实现了这些功能，但又不想花太多精力维护；现在要弄成通用的接口协议，那就没必要兼容它们了

之后如果有些大家公认还不稳定，或者字段还不好确定的接口，也许可以先标为试验性 API，等稳定了再改为公开 API

不过这只是我暂时的想法，可能考虑的不够多，欢迎讨论～
